Build_personal_portfolio

FarwaMuhibZada opened the pull request:
# CapstoneProject_Personal_Portfolio 🚀
## Project Description 📝
This project serves as a comprehensive showcase of my skills, experiences, and achievements in  web Development. This repository contains the code and assets for my personal portfolio website. It includes a collection of projects, case studies, and blog posts that demonstrate my expertise in web development. The portfolio is designed to provide an overview of my professional journey, highlighting my accomplishments and showcasing my capabilities.
This project consist of 4 sections:

1.Header:
The introduction section of this portfolio provides a concise overview of the creator's background, skills in short bio.

2.About ME
The introduction section of this portfolio provides a concise overview of the creator's background, skills, and professional journey. It serves as an introductory statement that sets the stage for what visitors can expect to explore further.

3.Projects 
This section features a curated selection of key projects that highlight the creator's expertise and capabilities. Each project is presented with a brief description, showcasing the problem-solving skills, creativity, and technical proficiency involved. It may include links to live demos, GitHub repositories, or case studies for a deeper dive into the project details.

4.contact Us
Contact Form: A contact form allows visitors to send messages directly from the portfolio website. It typically asks for their name, email address, and message. The form streamlines communication and ensures that inquiries are organized and easily manageable.
## NOTE: check the  responsiveness of website in 1440px for desktop version and 393px for mobile version.

## Figma Design Link✍
https://www.figma.com/design/BFSpaGK7qdPXPwpVFzVANk/Ptotofolio?node-id=0-1&t=qLuEfa9gaEB8Veiw-0

## Demo 📸
![Capture](https://github.com/user-attachments/assets/da37dccd-91cf-41fb-b7db-1680530e4dba)
#life demo link 
https://farwamuhibzada.github.io/CapstoneProject_Personal_Portfolio/
## Design Idea By  UX/UI Team: Parwana hussani👩‍💻 , MuzhganAlizada👩‍💻 ,ZahraSadat👩‍💻

FarwaMuhibZada committed:
added readme.md file.

FarwaMuhibZada committed:
added linters config files.

FarwaMuhibZada committed:
added index.html styles.css and script.js file.

FarwaMuhibZada committed:
added navbar section.

FarwaMuhibZada committed:
added navbar for moile screen.

FarwaMuhibZada committed:
added about section in index.html file.

FarwaMuhibZada committed:
complete the design section of about page.

FarwaMuhibZada committed:
fixed linters error .

FarwaMuhibZada committed:
fixed linters error .

FarwaMuhibZada committed:
fixed linters error .

FarwaMuhibZada committed:
added the project section and completed the design and functionality.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
complete the design of desktop version 1440px screen size.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
change pxcel unit into rem .

FarwaMuhibZada committed:
complete the design of  desktop version .

FarwaMuhibZada committed:
complete the design of  desktop version .

FarwaMuhibZada committed:
complete the design of  desktop version .

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
complete the design of mobile version .

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
complete  the desin modal of mobile version

FarwaMuhibZada committed:
complete  the desin modal for desktop  version

FarwaMuhibZada committed:
complete the design of desktop version

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
added deployment link .

FarwaMuhibZada committed:
added boder to the project images .

FarwaMuhibZada committed:
complete the design of website .

MasomRezaie commented:
@FarwaMuhibZada 
Dear Farwa, your project is perfect specially its design. nothing is missed. your code is clean, its responsive and the functionality also work. from my side.it is perfect. I approved your project.

FarwaMuhibZada commented:
> @FarwaMuhibZada Dear Farwa, your project is perfect specially its design. nothing is missed. your code is clean, its responsive and the functionality also work. from my side.it is perfect. I approved your project.

dear @MasomRezaie ! thanks for reviewing my project and your approval .

ahzamir commented on styles.css:
## Pop-up Modal

### Your Design:
![image](https://github.com/user-attachments/assets/21f5d7bb-7b70-4be5-821e-09ba5578fa7c)
|
![image](https://github.com/user-attachments/assets/69d0b709-e7eb-4680-89a3-f17fe8ab468c)

### Figma Design:
![image](https://github.com/user-attachments/assets/81c7db55-2142-4069-bf44-1d5a38348d78)
|
![image](https://github.com/user-attachments/assets/d50c5bb2-c327-4981-8d30-c3f765b7737d)


- The pop-up modal size for the desktop version is small so make it a little larger.
- Make your Pop-up modal centered on the page for the desktop and mobile versions, for now, it is showing on the top.
- The image size is smaller than the Figma design image size and does not look the same as the Figma design please make changes to it for both mobile and desktop versions.
- Font Issue (font is not the same as the Figma design font)
- Text-alignment issue (Not same as the figma, your design is centered but figma is left side).
- The space between elements on mobile version is different from the figma design make changes on it and make it same as the figma design.
- Add the languages used for every project on the pop-up modal same as the figma design
- The close icon on Figma design has a rounded border but I can not see it on your design please add it.

FarwaMuhibZada replied:
@ahzamir hello sir!
size of modal in figma design for desktop version is 1400px and for mobile it will cover all the page and make both horizontal and vertical scrollbar  what is your suggestion

ahzamir commented on styles.css:
## Toolbar (header):

### Your Design:
![image](https://github.com/user-attachments/assets/571644e1-d179-42a6-95c5-d8b70f51b870)
|
![image](https://github.com/user-attachments/assets/9e94dc3e-1270-49e6-a691-8f8a243182e4)

### Figma:
![image](https://github.com/user-attachments/assets/5a946ac5-631c-4402-805b-454e491df588)
|
![image](https://github.com/user-attachments/assets/82662e92-42b4-49be-807c-21f03061c9f4)

- The color for the toolbar links is different in the Figma Design with your design for both mobile and desktop versions.
- Add the borders between nav links same as the Figma design.
- The alignment on the figma design is left your's are centered.

ahzamir commented on styles.css:
## Home Page:

### Your Design:
![image](https://github.com/user-attachments/assets/bcde9e6d-aae8-4b1a-9d81-e62bf6b807cc)
|
![image](https://github.com/user-attachments/assets/decaf955-9ca7-4158-9e67-bab2e16ab922)

### Figma Design:
![image](https://github.com/user-attachments/assets/fde851ab-776d-4e27-85bf-3e8c2588306a)
|
![image](https://github.com/user-attachments/assets/75bc3096-da94-4b8c-a60c-240ede8ea3dc)

- The text sizing and font weight on the mobile version are not the same as the Figma design please make it the same as the Figma.

ahzamir commented on styles.css:
- Well done, about me page is same as figma design just the heading text font is not same change it.

ahzamir commented on styles.css:
## Projects page:

### Your Design:
![image](https://github.com/user-attachments/assets/29a55792-086f-41ee-83b2-2a3013037ab4)

### Figma Design:
![image](https://github.com/user-attachments/assets/616c9f7d-c3b3-460d-bbe3-f470415bef4f)

- The padding issue on both, the mobile version and the desktop version.
- The background color issue on the desktop version.
- Font issue.

ahzamir commented on styles.css:
## Contact Page:

### Your Design:
![image](https://github.com/user-attachments/assets/39a80c21-01e5-4bc7-a6e5-15e84c5702f4)
|
![image](https://github.com/user-attachments/assets/13cbb413-fda6-410f-b2f6-bd4202af5feb)

### Figma:
![image](https://github.com/user-attachments/assets/2a971873-edb1-468b-b2fe-2d93c15183b0)
|
![image](https://github.com/user-attachments/assets/0985d32f-01bb-43e0-8bbb-3c46ee10d4ca)

- The text alignment is different from the Figma design it's centered on the Figma and your's is left.
- The space between the form inputs are different from the Figma design.
- Submit button background color is different from the Figma design, also the alignment of inside text and the alignment of the button has issues and different from figma design.

ahzamir commented on README.md:
```diff
@@ -1,2 +1,103 @@
+# Zahra SADAT👩‍💻
+- LinkedIn: [(https://www.linkedin.com/in/zahra-sadat-42340a310)]
+# Muzhgan Alizada👩‍💻
+- LinkedIn: [(https://www.linkedin.com/in/muzhgan-alizada-314a72310?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app)]
```
- use 3 hashes ### for names as the one hash # is for creating headings.

ahzamir reviewed (changes requested):
### Hi @FarwaMuhibZada 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- I really appreciate your hard work, dedication, and passion on this project, your project's functionality is perfect and 100% the same as the requirements, but still your project design has some minor issues you need to work on in order to get the final approval.
  - Check the inline comments below for details.
  
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

![DespicableMeGruGIF](https://github.com/user-attachments/assets/4edf36de-413f-4a2c-9287-10336ad4351b)

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Personal Portfolio Functionality requirments.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
❌Your design is the same as the fFgma design

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

FarwaMuhibZada committed:
change the color of toolbar and aligned the item left for both mobile and desktop version..

FarwaMuhibZada committed:
solved linters error.

FarwaMuhibZada committed:
using 3# insted of 2# in readme.md file .

FarwaMuhibZada committed:
change the size of humburger menu .

FarwaMuhibZada committed:
solved linters error

FarwaMuhibZada committed:
solved linters error

FarwaMuhibZada committed:
solved linters error

FarwaMuhibZada committed:
changed the font family of about heading

FarwaMuhibZada committed:
change the alignment text of contact as

FarwaMuhibZada committed:
solved linters error

FarwaMuhibZada committed:
solved linters error

FarwaMuhibZada committed:
solved linters error

FarwaMuhibZada committed:
change the paddin design of boxes in my project section

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
change the size of modals.

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
change the ackground color of my project section .

FarwaMuhibZada committed:
added tools in to modals.

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
fixed linters error.

FarwaMuhibZada committed:
change the size of modal image .

FarwaMuhibZada commented:
@ahzamir  Hello sir!
hope you are doing well I applied all your feedback on my project, kindly request you to review my project.

ahzamir commented on styles.css:
[SUGESSION] - Make the text a little bit smaller on both sizes:

![image](https://github.com/user-attachments/assets/f56d1d94-12d0-4794-b50b-aa5b180410bd)

ahzamir reviewed (approved):
### Hi @FarwaMuhibZada 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
![image](https://github.com/user-attachments/assets/554bda42-8471-445a-8e4f-da6194d99227)

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

![WellDoneDespicableMeGIF](https://github.com/user-attachments/assets/82e4929f-8b46-455c-aebf-2ec7cea59603)

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)✔️ You have followed the Cash Register project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
✔️Your design following the exact design of the Figma design

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

FarwaMuhibZada committed:
changed the font size of about section  .

FarwaMuhibZada committed:
fixed linters error .

FarwaMuhibZada committed:
fixed the height of modal in mobile version

FarwaMuhibZada commented:
Hello @ahzamir!
I hope you are doing well first of  all thanks a lot for your valuable feedbacks it make my project more efficient , secondly  I changed the font size of about section and make it a little bit small and   also changed the height of modals to look good 
 on mobile version.  can I merge my branch ??

ahzamir commented:
> Hello @ahzamir! I hope you are doing well first of all thanks a lot for your valuable feedbacks it make my project more efficient , secondly I changed the font size of about section and make it a little bit small and also changed the height of modals to look good on mobile version. can I merge my branch ??

- Well done, you can merge your branch. ✔️🎯✅👍

ahzamir reviewed (approved):
- Approved
